gprofiler/main.py

```diff
@@ -504,7 +504,7 @@ def main():
     reset_umask()
 
     try:
-        logger.info(f"Running gprofiler (version {__version__})...")
+        logger.info(f"Running gprofiler (version {__version__}): {' '.join(sys.argv[1:])}")
```

Comment:
wrap with quotes,
```suggestion
        logger.info(f"Running gprofiler (version {__version__}): '{' '.join(sys.argv[1:])}'")
```
maybe even add a "command line" prefix, so it will be very explicit (that this is the command line)

Reply:
k